tutor/src/screens/assignment-edit/tasking.js

```diff
@@ -222,8 +220,9 @@ class Tasking extends React.Component {
               labelWrapper={ux.isShowingPeriodTaskings ? null : NewTooltip}
               name={`tasking_plans[${index}].closes_at`}
               onChange={this.onClosesChange}
-              disabledDate={this.course.isInvalidAssignmentDate}
+              disabledDate={disabledDate}
               timezone={this.course.timezone}
```

Comment:
Why'd this change?   `this.course.isInvalidAssignmentDate` should be doing the same logic I think?  If it's not correct we should update that method since that method is only used here.

Good catch on the bug where the prop was mistakenly called `disabled` 😁

Reply:
Yeah, I should have updated the logic in `isInvalidAssignmentDate`. The current logic does not disable dates before the course start date and after the course end date. I will update it.